src/Stan/Analysis/Pretty.hs

```diff
@@ -62,8 +67,49 @@ analysisToNumbers Analysis{..} = AnalysisNumbers
     , anIns        = HS.size analysisInspections
     , anFoundObs   = length analysisObservations
     , anIgnoredObs = length analysisIgnoredObservations
+    , anHealth     = calculatedHealth
     }
-
+  where
+    calculatedHealth :: Double
+    calculatedHealth =
+        -- all inspections ignored or no observations
+        if null analysisInspections || null analysisObservations
+        then 100
+        else
+            let totalInspections = fromIntegral $ HS.size analysisInspections
+                triggeredInspections =
+                    fromIntegral
+                    $ Set.size
+                    $ Set.fromList
+                    $ map observationInspectionId
+                    $ toList analysisObservations
+
+            in 100 * (1 - triggeredInspections / totalInspections)
+
+{- | Show project health as pretty text with 2 digits after dot.
+-}
+prettyHealth :: Double -> Text
+prettyHealth health =
+    if fromIntegral (floor health :: Int) == health  -- display without decimal part
+    then toText (printf "%.0f" health :: String) <> "%"
+    else toText (printf "%.2f" health :: String) <> "%"
+
+{- | Enum to describe project health depending on the value of
+'anHealth'.
+-}
+data ProjectHealth
+    = Unhealthy
+    | LowHealth
+    | MediumHealth
+    | Healthy
+
+-- | Calculate 'ProjectHealth'.
+toProjectHealth :: Double -> ProjectHealth
+toProjectHealth health
+    | health >= 100 = Healthy
+    | health >= 80  = MediumHealth
+    | health >= 40  = LowHealth
+    | otherwise     = Unhealthy
```

Comment:
Currently, this function is not used. I've implemented it to display health better later in HTML. Let me know, if it can be used somehow in CLI reports as well :slightly_smiling_face: